deps/udev.json

```diff
@@ -0,0 +1,43 @@
+{
+    "name": "udev",
```

Comment:
In https://github.com/flathub/shared-modules

Reply:
ok